topics/metagenomics/tutorials/cami2-assembly/tutorial.md

```diff
@@ -0,0 +1,1085 @@
+
+Dataset (315Gb) includes 3x21 (5 Gb each paired-end read) paired-end short (with the length of 150bp) and long (with the length of 7408 bp or 3000 bp (sd 1000 bp)) read sequences
+
+
+#### Dataset choice
+
+We analysed all datasets and compared them with each other in terms of number of samples, dataset size, average read length.
+
+> ### {% icon details %} Datasets comparison
+>
+> <iframe src="https://docs.google.com/spreadsheets/d/e/2PACX-1vQgJr3J-IyVy9IkXS9W-RZcV83Tr6f7RusG_97QwgpW2dFdCXUMroROIhy8gKjPcUgISFXW9NQwOzzK/pubhtml?gid=650574223&amp;single=true&amp;widget=true&amp;headers=false" width="900" height="600" frameborder="0" marginheight="0" marginwidth="0">Loading…</iframe>
+>
+{: .details}
+
+
+1. After close consideration of all datasets we decide to choose a marine dataset:It contains paired-end reads from 10 samples, both short (sequenced with Illumina method) and long (sequenced with nanopore method). ITmeans that we can try different tools on either one sample or pooled sample collection and also rub tools working with long reads and short reads, and tools working with both short and long at the same time. 
+2. Overall size of all marine data is 180 Gb. This size is big enough and allows us to test tools and measure such metrics as runtime and memory usage. Other datasets (strain madness and plant-associated) are too large, and since we have a limit in time of this tutorial we decided to give preference to smaller but highly representable marine dataset.
+
+
+### Get datasets
+
+So the assembly challenge is our choice of challenge that we are interested in. And marine dataset is our choice of dataset that we want to launch an assembly challenge on. Let’s start with the first step - uploading data to Galaxy.
+
+
+#### Download data from CAMI2 resources.
+
+Since data is of large size we cannot just download them to our computer by clicking the button download. Additionally, data cannot be downloaded at once, datasets are divided by samples - one file per one sample. There are also different files for long and short reads of the same sample.
+
+It is possible to download datasets using DOI links from [Table 2](https://www.nature.com/articles/s41596-020-00480-3/tables/2) CAMI Tutorial [[20]](https://www.nature.com/articles/s41596-020-00480-3)
+
+| CAMI benchmark dataset                                               | DOI                                              |
+|--------------------------------------------------------------------- |------------------------------------------------- |
+| CAMI I: low, medium, high complexity, and ‘toy’ datasets             | 10.5524/100344                                   |
+| CAMI II: human microbiome project and mouse gut toy datasets         | 10.4126/FRL01-006425518; 10.4126/FRL01-006421672 |
+| CAMI II: marine, strain madness, rhizosphere, and pathogen detection | 10.4126/FRL01-006425521                          |
```

Comment:
Here it'd be really nice to link those DOIs! Either using `{% cite %}` (+ add to bib file) or hyperlinking directly.